Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -18,16 +18,23 @@
 
 #include "LibFuzzer/FuzzerDefs.h"
 
+#include "Firestore/core/src/firebase/firestore/model/database_id.h"
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
+using firebase::firestore::model::DatabaseId;
 using firebase::firestore::remote::Serializer;
 
 namespace {
 
+// Global static objects. Initialized before fuzzing starts.
+static Serializer *serializer;
+
 // Fuzz-test the deserialization process in Firestore. The Serializer reads raw
 // bytes and converts them to a model object.
 void FuzzTestDeserialization(const uint8_t *data, size_t size) {
-  // TODO(minafarid): fuzz-test Serializer.
+  @try {
+    serializer->DecodeFieldValue(data, size);
+  } @catch (...) {}
```

Comment:
Please add a comment to make it clear why we don't care about exceptions when fuzzing.